api/v3/EventGroup.php

```diff
@@ -0,0 +1,56 @@
+<?php
+use CRM_MembersOnlyEvent_ExtensionUtil as E;
+
+/**
+ * EventGroup.create API specification (optional).
+ * This is used for documentation and validation.
+ *
+ * @param array $spec description of fields supported by this API call
+ *
+ * @see https://docs.civicrm.org/dev/en/latest/framework/api-architecture/
+ */
+function _civicrm_api3_event_group_create_spec(&$spec) {
```

Comment:
Perhaps, we should remove the function if it does nothing at all?

Reply:
I agree.